tensorflow/stream_executor/stream.cc

```diff
@@ -1886,7 +1886,7 @@ Stream *Stream::GetOrCreateSubStream() {
 
   // Look for the first reusable sub_stream that is ok, dropping !ok sub_streams
   // we encounter along the way.
-  for (int64 index = 0; index < sub_streams_.size();) {
+  for (int64 index = 0, end = sub_streams_.size(); index < end;) {
```

Comment:
You cannot do this here. `sub_streams_` is modified inside the loop, hence the `size()` is not constant, hence you cannot extract it to a separate local, hence this causes the segfault we're seeing in the Ubuntu failure

Reply:
Thank you very much for the note. 

I made an update.